src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -256,6 +272,38 @@ const MapView: React.FC<MapViewProps> = ({ blocks, neighborhoods, sites }) => {
           });
         });
 
+        // Creates a new layer
+        const sitesLayer = new google.maps.Data({ map });
+
+        // Loads the objects into the layer
+        sitesLayer.addGeoJson(sites);
+
+        function setSitesStyle(v: boolean) {
+          sitesLayer.setStyle((feature) => {
+            let visible = false;
+            let icon = treeIcon;
+
+            // Only shows sites if there is a tree there
+            if (feature.getProperty('tree_present')) {
+              visible = v && true;
+            }
+
+            // TODO: update this to if the tree was planted within the past three years
+            // If the tree has not been updated within the past three years, use youngTreeIcon
```

Comment:
Right now this is a png. Are we going to have different images for the different adoption statuses as well?

Reply:
Yeah, I'll add that now